docs/wiki/ALZ-Contribution-Guide.md

```diff
@@ -0,0 +1,183 @@
+
+Policy versioning follows the same protocol as built-in policies. More information on that can be found in the [ALZ Policies document in the wiki](./ALZ-Policies.md#versioning).
+
+For policy deprecation, the process is documented in the [Azure Landing Zones - Deprecating Policies](./ALZ-Deprecated-Services.md) page.
+
+If a policy is part of an initiative, references to policies that are being deprecated should be removed. Policy initiatives are located in the [policySetDefinitions](../../src/resources/Microsoft.Authorization/policySetDefinitions/) folder. To find out if a policy is part of an initiative it is recommended to look up the policy definition in [AzAdvertiser](http://azadvertizer.com/) and check for association with initiatives. When identified, go into the necessary initiative and remove references to the definition. Locate the policy definition in the parameters of the initiative and remove reference:
+
+![Example policy def in initiative](./media/example-def-in-init.png)
+
+Also find it in the policyDefinitions and remove reference as well:
+
+![Example policy def in initiative 2](./media/example-def-in-init-2.png)
+
+When working within the policy files, to read parameters which are set at the top level of the policy definition a double escape is needed for ARM. So instead of using `[parameters('someParameter')]` within the policy, you should use `[[parameters('someParameter')]` instead.
+
+> **Note:** When testing the policy manually in the portal or another deployment outside of the ALZ Accelerator (Portal), you will need to remove the double escaping, `[[`, and revert to normal ,`[`'
+
+When working with policies that are assigned by default, these are located under the [eslzArm/managementGroupTemplates/policyAssignments](../../eslzArm/managementGroupTemplates/policyAssignments) folder. References to policy definitions are done through the assignments, so if any amendments are done to default assigned policies, they should be amended here too. A wiki to default assignments can be found [in the wiki](./ALZ-Policies.md).
+
+Policies in `eslzArm.json` file will also need updating if wanting to assign a new policy that is located. The file for this amendment [is located here](../../eslzArm/eslzArm.json).
+
+Details of 
+
+### Contribution scope
+
+The following is the scope of contributions to this repository:
+
+As the Azure platform evolves and new services and features are validated in production with customers, the design guidelines will be updated in the overall architecture context.
+
+With new Services, Resources, Resource properties and API versions, the implementation guide and reference implementation must be updated as appropriate.
+Primarily, the code contribution would be centered on Azure Policy definitions and Azure Policy assignments for the reference implementation.
+
+Submit a pull request for documentation updates using the following template 'placeholder'.
+
+### How to submit Pull Request to upstream repo
```

Comment:
Is it worth renaming and expanding this section to cover "Developer workflow", starting with "Clone the repository" and working through to "Raising a PR"?

Reply:
Or link to some good github guide that must exist for this?

Reply:
@krowlandson @jtracey93 

I've added links to existing guidance and renamed the section. PR should now be amended to comments accordingly